tools/clang/unittests/HLSLHost/CMakeLists.txt

```diff
@@ -11,16 +11,10 @@ add_clang_executable(HLSLHost
   HLSLHost.cpp
   )
 
-target_link_libraries(HLSLHost
-  ClangHLSLTests
-  )
-
 set_target_properties(HLSLHost PROPERTIES VERSION ${CLANG_EXECUTABLE_VERSION})
 
 include_directories(AFTER ${DIASDK_INCLUDE_DIRS})
 
-add_dependencies(HLSLHost ClangHLSLTests)
-
```

Comment:
I presume this fixed the odd failure in automated builds. Why was this an invalid dependency?